### ISSUE TITLE
Export data (in query) as an excel file.

### ISSUE THREAD

**Issue by bvizureanu** [code4romania/monitorizare-vot#208]
```sql
       SELECT
			obs.Phone,
			obs.IdNgo,
			f.Code as FormCode,
			q.Text as Question,
			o.Text as [Option],
			a.[Value] as [FreeText],
			n.Text as Note,
			n.AttachementPath,
			a.LastModified,
			a.CountyCode,
			a.PollingStationNumber
		FROM 
			(Answers a 
			INNER JOIN Observers obs
				ON a.IdObserver = obs.Id
			INNER JOIN OptionsToQuestions oq
				ON a.IdOptionToQuestion = oq.Id
			INNER JOIN Options o 
				ON oq.IdOption = o.Id
			INNER JOIN Questions q
				ON oq.IdQuestion = q.Id
			INNER JOIN FormSections fs
				ON q.IdSection = fs.Id
			INNER JOIN Forms f
				ON fs.IdForm = f.Id)
			LEFT JOIN Notes n
				ON n.IdQuestion = q.Id AND n.IdObserver = obs.Id AND n.IdPollingStation = a.IdPollingStation
		WHERE
			a.LastModified >= '2019-11-08 06:00:00'
			AND a.IdObserver > 10
			AND obs.IdNgo <> 1
```

accept filters:
- NGO
- Observer
- PollingStation
- County
- Form
- Period (from -> to | datetime)

The data will be pivoted; redundancy is good.

**Comment by idormenco** [comment]
On it

**Comment by RaduCStefanescu** [comment]
Fixed in #212 
The issue still needs some more adjustments so I will leave this one open.

**Comment by bvizureanu** [comment]
this is done from the api's perspective
